microsoft/testsuites/dpdk/dpdktestpmd.py

```diff
@@ -179,7 +183,7 @@ def _install_dependencies(self) -> bool:
 
     def _generate_testpmd_command(self, nic_to_include: str) -> str:
         return (
-            f"{self._testpmd_install_path} -l 2,3 -n 4 --proc-type=primary "
+            f"{self._testpmd_install_path} -l 0-1 -n 4 --proc-type=primary "
```

Comment:
Add comments to explain the magic numbers.

Reply:
added